src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -1146,6 +1146,17 @@ void uct_rc_mlx5_iface_common_sync_cqs_ci(uct_rc_mlx5_iface_common_t *iface,
 #endif
 }
 
+ucs_status_t uct_rc_mlx5_create_cq(uct_ib_iface_t *iface, uct_ib_dir_t dir,
+                                   const uct_ib_iface_init_attr_t *init_attr,
+                                   int preferred_cpu, size_t inl)
+{
+    uct_rc_mlx5_iface_common_t *mlx5_iface =
```

Comment:
can we pass mlx5_iface to this func as a argument?

Reply:
No we can't, because we call this function during the IB iface initialization.